### PR TITLE
RATEST-108: Fix Travis failure on reference application ui-tests module

### PR DESCRIPTION
**Pull request by kdaud**
Ticket ID: https://issues.openmrs.org/browse/RATEST-108
Description:
Fixed travis failure on _reference application ui-tests_ module

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@kdaud  our role now we well be revisiting the tests ignored for each file so as to have them remotely passing

**Comment by kdaud** [comment]
> @kdaud our role now we well be revisiting the tests ignored for each file so as to have them remotely passing

I agree. This suggestively means prioritizing on resurrecting these tests. cc: @kaweesi

**Comment by kaweesi** [comment]
I don't think we have failing tests to ignore, https://github.com/openmrs/openmrs-distro-referenceapplication/tree/master/ui-tests

**Comment by kaweesi** [comment]
master should be passing,

**Comment by HerbertYiga** [comment]
> master should be passing,

For the different tests we have been making, travis has been breaking with a list of tests that have been ignored in this pr

**Comment by kdaud** [comment]
> @kdaud our role now we well be revisiting the tests ignored for each file so as to have them remotely passing

I agree. This suggestively means prioritizing on resurrecting these tests. cc: @kaweesi 

> I don't think we have failing tests to ignore, https://github.com/openmrs/openmrs-distro-referenceapplication/tree/master/ui-tests



> I don't think we have failing tests to ignore, https://github.com/openmrs/openmrs-distro-referenceapplication/tree/master/ui-tests

Actually testing them locally without ignore annotation in the dev environment, these tests are fail. @HerbertYiga do you mind testing any of these and running it as JUnit test within the IDE and share the experience

**Comment by kaweesi** [comment]
ensure u are using the same dataset, these tests and tied into a dataset, for-example searching a patient by name you don't have locally would fail the find patient test

**Comment by HerbertYiga** [comment]
> Actually testing them locally without ignore annotation in the dev environment, these tests fail. @HerbertYiga do you mind testing any of these and running it as JUnit test within the IDE and share the experience

@kdaud  try running NamePatientAccentedLetterTes it runs well at my side

**Comment by HerbertYiga** [comment]
> ensure u are using the same dataset, these tests and tied into a dataset, for-example searching a patient by name you don't have locally would fail the find patient test

And to  add on this, the tests first see the data you have in your local instance before picking up on their own created data set,for example removing the patients in your local  instance will make sure that the tests use their  own data sets  for a given patient

**Comment by kdaud** [comment]
> @kdaud try running NamePatientAccentedLetterTes it runs well at my side

Screenshot at 2021-03-19 12-26-10

**Comment by HerbertYiga** [comment]
@kdaud  do you have any patient existing in your local instance

**Comment by kdaud** [comment]
> ensure u are using the same dataset, these tests and tied into a dataset, f

Definitely here is the catch from my side then. However, how is this achievable. @HerbertYiga

**Comment by kdaud** [comment]
> do you have any patient existing in your local instance

Hope you mean local server that is sited on my pc. If this is true, I have some patient registered already

**Comment by HerbertYiga** [comment]
> > ensure u are using the same dataset, these tests and tied into a dataset, f
> 
> Definitely here is the catch from my side then. However, how is this achievable. @HerbertYiga

what i do is that i dont leave any data in my instance,ie i delete the existing patient and rely on data sets from the  testing classes

**Comment by sherrif10** [comment]
https://github.com/openmrs/openmrs-distro-referenceapplication/pull/340#issuecomment-802678096
This makes sense thanks everyone

**Comment by gracepotma** [comment]
@kdaud some feedback for you. 
It seems like this PR is mostly about ignoring or disabling the failing tests. Ignoring tests is not how we actually fix build failures.
Please update the title & message of this commit to be more accurate, e.g. the commit message should have said something like “Ignoring all failing tests” or “Disabling all failing tests".  (I actually double checked this with @dkayiwa because I was originally confused at how ignoring a test could be the same thing as fixing a build.)

Does that make sense?

**Comment by kdaud** [comment]
> It seems like this PR is mostly about ignoring or disabling the failing tests. Ignoring tests is not how we actually fix build failures.
Thanks @gracepotma !!. Actually this PR came as a result of this [talk](https://talk.openmrs.org/t/ci-jobs-fails-on-openmrs-distro-referenceapplication-uitests-module/32742/18) post as we thought to be a temporal solution to failing jobs that qa contributors were encountering with the module. However, @kaweesi gave directives to address the concern rather than this pr

**Comment by kdaud** [comment]
> Please update the title & message of this commit to be more accurate, e.g. the commit message should have said something like “Ignoring all failing tests” or “Disabling all failing tests". (I actually double checked this with @dkayiwa because I was originally confused at how ignoring a test could be the same thing as fixing a build.)

I agree

File: ui-tests/src/test/java/org/openmrs/reference/AddDiagnosisToVisitNoteTest.java [hunk 1]
```diff
@@ -36,6 +36,7 @@ public void setup() {
         createTestVisit();
     }
 
+	@Test
     @Ignore
     @Category(BuildTests.class)
     public void AddDiagnosisToVisitNoteTest() {
```
Comment (lyndseyBeil) [inline ui-tests/src/test/java/org/openmrs/reference/AddDiagnosisToVisitNoteTest.java:42]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/AddDiagnosisToVisitNoteTest.java:42]: > I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
> Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

@lyndseyBeil Thanks for this cache. Its a good idea if you create a ticket for this and go a head to work on it

**Pull request closed without merging**